nuxt.config.ts

```diff
@@ -18,8 +17,13 @@ export default defineNuxtConfig({
   ],
   nitro: {
     prerender: {
-      crawlLinks: false
-    }
+      crawlLinks: true,
+      routes: ['/en/', '/nl/'],
+    },
+  },
+  routeRules: {
+    '/**': { swr: true },
+    '/*/team/**': { prerender: false },
```

Comment:
What's the reason for server side rendering the team page? Didn't we want to switch to nuxt build while keeping the prerender for every page for now?

Reply:
Yes we can, I wanted to test if I could stop the crawler from pre-rendering some pages so I chose the team pages since I know that's a lot of pages which don't get visited often. I kinda want to keep it since it shortens the build time and gives us a taste to see how those pages perform with swr/isr.